com.unity.render-pipelines.high-definition/Documentation~/whats-new-11.md

```diff
@@ -20,6 +20,14 @@ Cubemap fields now accept both [RenderTextures](https://docs.unity3d.com/Manual/
 
 For more information, see the [HDRI Sky](Override-HDRI-Sky.md) and [Physically Based Sky](Override-Physically-Based-Sky) sections of the documentation.
 
+### Density Volume Improvements
+
+Density Volumes masks has been reworked to allow 3D RenderTextures to be used as masks. 3D Mask textures now uses all four RGBA channel which allows volumetric fog to have different colors and density based on the 3D Texture.
```

Comment:
Density Volumes masks now support using 3D RenderTextures as masks. 3D mask textures now also use all four RGBA channel which allows volumetric fog to have different colors and density based on the 3D Texture.